tests/bluetooth/controller/common/src/helper_pdu.c

```diff
@@ -504,6 +504,33 @@ void helper_pdu_encode_sca_rsp(struct pdu_data *pdu, void *param)
 	pdu->llctrl.clock_accuracy_rsp.sca = p->sca;
 }
 
+void helper_pdu_encode_periodic_sync_ind(struct pdu_data *pdu, void *param)
+{
+	struct pdu_data_llctrl_periodic_sync_ind *p = param;
+
+	pdu->ll_id = PDU_DATA_LLID_CTRL;
+	pdu->len = offsetof(struct pdu_data_llctrl, periodic_sync_ind) +
+		sizeof(struct pdu_data_llctrl_periodic_sync_ind);
+	pdu->llctrl.opcode = PDU_DATA_LLCTRL_TYPE_PERIODIC_SYNC_IND;
+
+	pdu->llctrl.periodic_sync_ind.id = sys_cpu_to_le16(p->id);
+
+	memcpy(&pdu->llctrl.periodic_sync_ind.sync_info, &p->sync_info,
+			sizeof(struct pdu_adv_sync_info));
+
+	pdu->llctrl.periodic_sync_ind.conn_event_count = sys_cpu_to_le16(p->conn_event_count);
+	pdu->llctrl.periodic_sync_ind.last_pa_event_counter =
+							sys_cpu_to_le16(p->last_pa_event_counter);
+	pdu->llctrl.periodic_sync_ind.sid = p->sid;
+	pdu->llctrl.periodic_sync_ind.addr_type = p->addr_type;
+	pdu->llctrl.periodic_sync_ind.sca = p->sca;
+	pdu->llctrl.periodic_sync_ind.phy = p->phy;
+
+	memcpy(pdu->llctrl.periodic_sync_ind.adv_addr, p->adv_addr, sizeof(p->adv_addr));
+
+	pdu->llctrl.periodic_sync_ind.sync_conn_event_count = p->sync_conn_event_count;
```

Comment:
Consider storing `pdu->llctrl.periodic_sync_ind` as a local variable to reduce line lengths

Reply:
However, if this is changed it will not be consistent with the rest of the code in `helper_pdu.c`.

Reply:
Welp :) 

It's OK to keep as is